### DISCUSSION TITLE
tiles default margin

### DISCUSSION THREAD

**Discussion by AZ77-Git** [olton/Metro-UI-CSS#1821] [Q&A]
hello,

I have noticed that the tiles have automatically a margin.

here : https://codepen.io/AZ77/pen/MWmQEpP

at the right of tiles 2, 3, and 6, there is a white margin.

How can I remove it ? 

thank you for your help.

**Comment by AZ77-Git** [comment]
Hello,

Have you any idea ? 

Thanks.

**Comment by olton** [comment]
```
.tiles-grid.size-3 {
  width: 475px
}
```

**Comment by AZ77-Git** [comment]
the codepen was updated with your solution.

but nothing changed.

Have-you tested you solution in my codepen link ?

**Comment by olton** [comment]
```
.tiles-grid.size-3 {
  width: 475px!important;
}
```
